### ISSUE TITLE
"please 'upgrade' to Windows 10" annoyance that doesn't go away

### ISSUE THREAD

**Issue by RedSteel-1** [mullvad/mullvad-browser#325]
I'm not going to downgrade to malwaredows 10, however, this nonsense notification continues to pop up, no matter how many times I close it.
Every time I restart the browser I see it again.
This is Very Annoying (and the notification text is Very Pissing-Off).
Kindly do something with this and make it go away.
Thanks

**Comment by ruihildt** [comment]
Closing as a duplicate of #323. (Also see #326 )

**Comment by Thorin-Oakenpants** [comment]
> no matter how many times I close it

use the button to acknowledge it, not the x to dismiss the nofification

**Comment by ruihildt** [comment]
@RedSteel-1 Which OS are you using?

**Comment by Thorin-Oakenpants** [comment]
> Already done. Doesn't work.

I have a bare metal windows 7 machine. When we first added the deprecation message, I got this notification message, like you. I actually have multiple TB's (stable, 32bit, alpha etc). I successfully clicked, from memory, the `OK` to acknowledge it, for each TB. But on one I was surprised that it came up again, and realized I must have hit the `x` instead - which is how I came to realize this is how it worked. I was able to still see the button and click OK to dismiss it for good.

That said, I haven't booted or updated my win7 for a wee while, and we've had updates. So I will retest. There is a pref that gets set when you want it permanently dismissed.

The other thing is the message has changed over time as Mozilla took a while to decide on ESR115 support in win7/8. So it may be that the message was deliberately re-activated by code to ensure users got the new message (IDK, am guessing).

tl;dr: I can easily test on a win7 machine when I get time, including downloading and testing old TB versions

cc: @PieroV fyi

**Comment by PieroV** [comment]
That message comes out after every update.

Anyway, Mullvad Browser has never officially supported Windows 7.

**Comment by Thorin-Oakenpants** [comment]
OK, so @RedSteel-1 the message will be enabled each update, so once a month :) Maybe this is what is confusing the issue. Otherwise, you _should_ be able to dismiss it (until the next update). I'll re-test in my windows 7 later on

**Comment by PieroV** [comment]
The logic is to display the message after each update, so it's normal you got this message.
We don't know if MB is going to get further updates, as the plan is to continue with 14.0 in 10-15 days.

**Comment by ruihildt** [comment]
We have never supported Mullvad Browser on Windows 7, I changed my incorrect tagging from ’duplicate’ to ’wont fix’.

We will be happy to provide support for any supported OS.
